noxfile.py

```diff
@@ -100,12 +100,11 @@ def build_docs(session: nox.Session) -> None:
     session.install("sphinx")
     session.install("sphinx-view")
     session.install("sphinx-autoapi")
-    session.run(
-        "sphinx-build",
-        "-W",
-        "docs",
-        "docs/_build",
-    )
+    session.install("sphinx-rtd-theme")
+    session.install("myst-parser")
+
+    session.chdir("./docs")
+    session.run("make", "html")
```

Comment:
If you use "make html" here, worth adding `-W` to `SPHINXOPTS ` in https://github.com/astronomer/astro-sdk/blob/2bbe5d09ddc8cdcf8fcd87e3788d7f579c886d34/docs/Makefile#L6 so that all the Sphinx warnings are taken care of to avoid docs with not-so-good formatting